File: Backend/RoomMe/RoomMe.API/Models/ScheduleListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ScheduleListModel
    {
        public int Id { get; set; }
        public string HouseworkName { get; set; }
        public DateTime Date { get; set; }
        public HouseworkStatusModel Status { get; set; }
        public int UserId { get; set; }
        public string UserName { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/schedule/ScheduleListModel.kt
package uj.roomme.domain.schedule

import uj.roomme.domain.housework.HouseworkStatusModel
import java.time.OffsetDateTime

data class ScheduleListModel(
    val id: Int,
    val houseworkName: String,
    val date: OffsetDateTime,
    val status: HouseworkStatusModel,
    val userId: Int,
    val userName: String
)
